mfr/extensions/ipynb/templates/viewer.mako

```diff
@@ -1,5 +1,5 @@
 <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,300,700" rel="stylesheet" type="text/css">
-<link rel="stylesheet" href="${base}/css/ipynb.css">
+<link rel="stylesheet" href="/assets/ipynb.css">
```

Comment:
Need to finish this. location of /static/css/default does'nt make sense to be in mfr/server, and is used as a dep in multiple extensions so need to decide on better location. Thinking /styles or /src/styles as is convention with other js apps